services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -262,26 +247,13 @@ class FlowHelper {
         uniqueTransitSwitches.each { node ->
             def rules = northbound.getSwitchRules(node.switchId).flowEntries.findAll {
                 !Cookie.isDefaultRule(it.cookie)
+            }*.cookie
+            if (mainFlowPath.find { it.switchId == node.switchId }) {
```

Comment:
Can we do 2 separate loops for `protectedFlowTransitSwitches.findAll { !commonSwitches.contains(it.switchId) }` and `mainFlowTransitSwitches.findAll { !commonSwitches.contains(it.switchId) }` instead of putting this if here? It complicates things

Reply:
done